Tella/Scenes/Reports/View Model/ReportsViewModel.swift

```diff
@@ -85,6 +83,41 @@ class ReportsViewModel: ObservableObject {
             }.store(in: &subscribers)
     }
     
+    private func updateDraftReportsNumber() {
```

Comment:
updateDraftReportsNumber, updateOutboxReportsNumber and updateSubmittedReportsNumber are not use , could you remove them